feat: add `once` decorator to `discord.Client` (#1940)
Pycord-Development/pycord: CHANGELOG.md

- Added new events `on_bridge_command`, `on_bridge_command_completion`, and
  `on_bridge_command_error`.
  ([#1916](https://github.com/Pycord-Development/pycord/pull/1916))

> - Added a `@client.once()` decorator that works like a one-time event listener.

Reviewer: ```suggestion
- Added the `@client.once()` decorator, which serves as a one-time event listener.
```
